np.sh

```diff
@@ -15,7 +15,7 @@ if test "0" != "$(git rev-list --count --left-only @'{u}'...HEAD)"; then
 	exit 128;
 fi
 
-trashCli=$(node -e "var path = require('path');console.log(path.join(path.dirname(require('fs').realpathSync('$0')), 'node_modules/.bin/trash'))");
+trashCli=$(node -e "console.log(require.resolve('.bin/trash'))");
```

Comment:
Oh clever. I didn't realize `require.resolve` worked with non-packages.

Can actually be simplified further:

```sh
trashCli=$(node -p "require.resolve('.bin/trash')");
```